### DISCUSSION TITLE
Welcome to NativePHP Discussions!

### DISCUSSION THREAD

**Discussion by simonhamp** [NativePHP/laravel#115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you're doing or hope to do with NativePHP.

**Comment by Giuliano1993** [comment]
Hi everyone! I'm Giuliano, a full-stack developer from Italy. 👋 
I love developing websites, web apps, and tools, but I've always been looking for a tool that allows me to take the development of the web to a local level; I tried many solutions, but being mainly a PHP developer, NativePHP has been for me some groundbreaking news! I'm still doing some small local experiments, but I'm looking forward to creating some tools for helping developers with it. I am looking forward to contributing to the package itself; it would be great for me to do so!
So we'll see around! It's fantastic to be here with other passionate developers 😃

**Comment by bbredewold** [comment]
Hi! 👋  I'm a Laravel developer from the Netherlands. 
Utilizing the TALL stack we work on projects for different clients. Besides work, I have - as all of you have - a bunch of side projects to play with, when we have some time left ;)

I'm using NativePHP to write a tool that alerts me when my glucose values are too high or low. As NativePHP is a desktop tool, I can use notifications and popups that don't rely on my webbrowser. So that's great for my use case :)

**Comment by mrchenzifan** [comment]
nice ~ NativePHP so cool~

**Comment by kisssg** [comment]
Cool
